website/docs/verifiable-document/did/create.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Creating a DID
+sidebar_label: Creating a DID
+---
+
+While there exists many [DIDs](/docs/appendix/glossary#did), this tutorial will focus only on `ethr` DID. Feel free to use any DID that uses public key of type `Secp256k1VerificationKey2018`, which is the only type supported by OpenAttestation at the moment.
```

Comment:
we can really use any DID with secp256k1? i didnt know that

Reply:
let's limit it to ethr for the moment :)